src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name, String color, double radius) {
+        super(name, color);
+        this.radius = radius;
+        setArea();
+    }
+
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
```

Comment:
```suggestion

    @Override
    public double getArea() {
        return Math.PI * radius * radius;
    }
```

And add the AreaCalculator

```
public interface AreaCalculator {
    double getArea();
}
```